sunpy/net/helioviewer.py

```diff
@@ -77,6 +78,8 @@ def get_closest_image(self, date, observatory=None, instrument=None,
                           detector=None, measurement=None, source_id=None):
         """
         Finds the closest image available for the specified source and date.
+        We can use `observatory` and `measurement` or `instrument` and `measurement` to get the value for source ID which
```

Comment:
Actually in fact, I wonder if wrapping this in a rst .. note:: would be better.

Reply:
Yeah I guess that would be better. I will change it

Reply:
I need to build the docs to see if it is getting rendered properly right?

Reply:
Yes.